addons/website_forum/models/forum.py

```diff
@@ -986,7 +986,10 @@ def _search_get_detail(self, website, order, options):
         }
 
         domain = website.website_domain()
-        domain += [('parent_id', '=', False), ('state', '=', 'active'), ('can_view', '=', True)]
+        domain += [('state', '=', 'active'), ('can_view', '=', True)]
```

Comment:
can you use `expression.AND` ? 'if you have "or" in "website_domain" you might have a wrong domain)

Reply:
I'm not sure what you mean here, what exactly is the problem?

Reply:
if `website_domain` is `['|', ('a', '=', 'a'), ('b', '=', 'b')]`, you will have `['|', ('a', '=', 'a'), ('b', '=', 'b'), ('state', '=', 'active'), ('can_view', '=', True)]` which will be `('a', '=', 'a') or [('b', '=', 'b'), ('state', '=', 'active'), ('can_view', '=', True)]` if I'm correct.

Moreover, now we (almost) always want to use expression.AND, expression.OR,  etc... it makes the code more explicit